package/MDAnalysis/topology/PrimitivePDBParser.py

```diff
@@ -167,10 +175,9 @@ def _parsebonds(self, atoms):
 
         bonds = set()
         with openany(self.filename, "r") as f:
-            lines = (line[6:].split() for line in f
-                     if line[:6] == "CONECT")
-            for bond in lines:
-                atom, atoms = int(bond[0]), map(int, bond[1:])
+            lines = (line for line in f if line[:6] == "CONECT")
```

Comment:
I understand why there is a check in `parse_conect()`: because it's a public function someone *might* think they can use it themselves and parse any old lines. It's good defensive programming.

One could replace the check below with an `assert` but I feel we're reduced to splitting hairs---assertion or proper exception.

If you decide to drop the check in `parse_conect()` then I would also add an underscore and make it `_parse_conect()`.